### PR TITLE
Fix autoload buffers

### PR DESCRIPTION
**Pull request by olimorris**
Hey @akinsho. Made a first attempt at #46.

I'm getting the same issue as #12, whereby when it autoloads, no syntax highlight is visible in the first buffer until you do `:e`. No idea why this is the case

### CODE REVIEW

**Comment by olimorris** [comment]
Alright we should be close to fully working now. This resolved the issues on my end. I moved the autocmds to the `plugins/persisted.lua` file which makes a whole lot more sense.

I just need to check if the ignoring of directories for autoloading still works as the test is failing for that.

File: lua/persisted/init.lua [hunk 1]
```diff
@@ -117,7 +121,7 @@ end
 ---@return nil
 function M.start()
   vim.api.nvim_create_autocmd(config.options.command, {
-    group = vim.api.nvim_create_augroup("Persisted", { clear = true }),
+    group = vim.api.nvim_create_augroup("Persisted", { clear = false }),
```
[the remark this answers is not in the document]
Comment (olimorris) [reply lua/persisted/init.lua]: You're spot on! Beautiful explanation too. Thank you

**Comment by olimorris** [comment]
@akinsho this should be good to go now. If you could have a final test and let me know 👍🏼

**Comment by akinsho** [comment]
Just tested it locally and it works like a charm. Thanks @olimorris

**Pull request merged**